docker-compose.yml

```diff
@@ -10,6 +10,5 @@ services:
       SUPABASE_SERVICE_ROLE_KEY: ${SUPABASE_SERVICE_ROLE_KEY}
       PASSWORD_PROTECTED: ${PASSWORD_PROTECTED}
       GRAFANA_PASSWORD: ${GRAFANA_PASSWORD}
-    platform: linux/amd64
```

Comment:
it is probably better to keep this in as Prometheus does not provide builds for mac and this tells docker it is ok to emulate